docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,654 @@
+---
+title: Integrating a React Native view into existing native apps
+sidebar_title: Integrated approach
```

Comment:
```suggestion
sidebar_title: Get started
```

Reply:
let's just call this get started because it's our default approach and people aren't ready to decide anything yet.